libnmstate/state.py

```diff
@@ -258,11 +277,22 @@ def merge_route_config(self, other_state):
                     self_iface_route_sets[route[Route.NEXT_HOP_INTERFACE]].add(
                         RouteEntry(route))
 
+        # Check whether user desire routes to down interface/ip.
```

Comment:
This comment should be  moved to the function docstring since it explains the function.